src/components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/V2V3PayForm.tsx

```diff
@@ -62,24 +59,12 @@ export const V2V3PayForm = ({
   const canAddMoreStickers =
     (stickerUrls ?? []).length < ProjectPreferences.MAX_IMAGES_PAYMENT_MEMO
 
-  useEffect(() => {
-    if (transactionCanceled) return
-
-    const initialStickerUrls = nftRewardTiers?.map(
-      (tier: NftRewardTier) => tier.fileUrl,
-    )
-
-    form.setFieldsValue({
-      stickerUrls: initialStickerUrls,
-    })
-  }, [form, nftRewardTiers, transactionCanceled])
-
-  useEffect(() => {
-    if (!form.getFieldValue('beneficiary')) return
-
-    setCustomBeneficiaryEnabled(transactionCanceled)
-  }, [form, transactionCanceled])
-
+  useEffect(
```

Comment:
this is pretty inconsequential, but we should add `beneficiaryEnabled` to the form prob.